Makefile

```diff
@@ -13,7 +8,7 @@ devnet: poetry-install
 
 # build
 build:
-	cargo build --all --release
+	cargo build -q --all --release
```

Comment:
why add the --quiet flag?

Reply:
It was there in kakrotup, so I updated my implementation accordingly, I can remove it, lemme know if you think it should be there and I will update.

Reply:
you can remove it:)

Reply:
done ser 🚀